src/internal/pachtmpl/pachtmpl.go

```diff
@@ -0,0 +1,105 @@
+package pachtmpl
+
+import (
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/google/go-jsonnet"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+)
+
+// ParseArgs parses args of the form key=value
+func ParseArgs(argStrs []string) (map[string]string, error) {
+	ret := make(map[string]string)
+	for _, argStr := range argStrs {
+		kv := strings.SplitN(argStr, "=", 2)
+		if len(kv) != 2 {
+			return nil, errors.Errorf("invalid template argument %q: must have form \"key=value\"", argStr)
+		}
+		key, value := kv[0], kv[1]
+		ret[key] = value
+	}
+	return ret, nil
+}
+
+// RenderTemplate renders the template tmpl, using args and returns the result.
+func RenderTemplate(tmpl string, args map[string]string) (string, error) {
+	vm := newVM(nil)
+	for key, value := range args {
+		vm.TLAVar(key, value)
+	}
+	output, err := vm.EvaluateAnonymousSnippet("main", string(tmpl))
+	if err != nil {
+		return "", errors.Wrapf(err, "template err")
+	}
+	return output, nil
+}
+
+// Eval evaluates the jsonnet at entrypointPath using fsContext to resolve imports
```

Comment:
A little unclear to me what the arguments are meant to be here and whether we're using this function. What's the use-case we're addressing here? Also what's the 'fs' in 'fsContext'?

Reply:
It stands for filesystem.  It's similar to a docker context, the idea is that you have a set of jsonnet files in the local filesystem and you want to evaluate the entrypoint, which could reference the other files through import statements.  

We still have to figure out the best way to collect the relevant files and send them over the wire.  That's not done yet, but if and when we figure out they, you would plug in the files there.

Reply:
Explained another way: The map is from file paths to file contents, and the entrypoint path is the path with the contents to evaluate.  When evaluating the contents of the file at entrypoint, import statements may result in additional lookups in the map.